source/config/configFlags.py

```diff
@@ -97,6 +97,25 @@ def _displayStringLabels(self):
 		}
 
 
+@unique
+class BrailleMode(DisplayStringStrEnum):
```

Comment:
I'm not sure whether we should introduce a new setting here. Is there anything against having a fourth tether setting? Tether automatically, focus, review and speech?